gradle.properties

```diff
@@ -15,4 +15,6 @@
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+android.enableJetifier=true
```

Comment:
If possible, try to run it today in 2021. without Jetifier. 
If it fails, still OK. 
If it works, yay, shorter build time.

Reply:
Works without it! 👍